ocrd_monitor/ocrdbrowser/_subprocess.py

```diff
@@ -24,7 +23,7 @@ def address(self) -> str:
         # (we use 8085 as fixed start of the internal port range,
         #  and map to the runtime corresponding external port)
         localport = self._localport.get()
-        return str(int(self._port) + localport - 8085)
+        return "http://localhost:" + str(int(self._port) + localport - 8085)
```

Comment:
It's strange to me that we still do this trick here. This maps the internal port range to a parallel external port **range**. But with the reverse proxy, the external port should be fixed now, right?